module-2/backend/week_three.md

```diff
@@ -10,16 +10,68 @@ Note: When you're done, submit a PR with a reflection in the comments about how
 ### Questions
 
 1. What are the 3 main features in an ERD?
+```
+Entities: Objects that we store the information about.
+Relationships: The way each entities are linked to one another
+Cardinalities: An attribute of each relationship, can either be "One to One", "One to Many", or "Many to Many"
+```
 2. Create an ERD for the following database: Restaurants, Customers, Items, Ingredients, Beverages, Orders, Vendors.
+```
+
+```
 3. What is the entry at the command line to create a new rails app?
+```
+rails new app_name
+```
 4. What do Models generally inherit from in rails?
+```
+Models inherit from ActiveRecord::Base
+```
 5. What do Controllers generally inherit from in a rails project?
+```
+Controllers inherit from ActionController::Base
+```
 6. How would I create a route if I wanted to see a specific horse in my routes fitle assuming I'm sticking to standard conventions and that I didn't want other CRUD functionality?
+```
+resources :horses, only [:show]
+```
 7. What rake task is useful when looking at routes, and what information does it give you?
+```
+'rake routes' prints the list of routes that have been defined in the project's routes file.
```

Comment:
what other information does it give you? it gives you the prefix, URI pattern, verb, controller & action